doc/network-zones.md

```diff
@@ -4,57 +4,149 @@ discourse: 12033,13128
 
 ## Add a network zone to a network
 
-This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4` and `dns.zone.reverse.ipv6` in network configuration.
+To add a zone to a network, set the corresponding configuration option in the network configuration:
+
+- For forward DNS records: `dns.zone.forward`
+- For IPv4 reverse DNS records: `dns.zone.reverse.ipv4`
+- For IPv6 reverse DNS records: `dns.zone.reverse.ipv6`
+
+For example:
+```bash
+lxc network set <network_name> dns.zone.forward="lxd.example.net"
+```
 
 Zones belong to projects and are tied to the `networks` features of projects.
-It is possible to restrict projects to specific domains and sub-domains through the `restricted.networks.zones` project configuration key.
+You can restrict projects to specific domains and sub-domains through the `restricted.networks.zones` project configuration key.
 
 ## Add custom records
 
-It's possible to add additional records to a zone.
-On the CLI, this is done through `lxc network zone record`.
+A network zone automatically generates a DNS record for each instance.
+If required, you can manually add custom records to a zone.
+
+To do so, use the `lxc network zone record` command.
+
+### Create a record
+
+Use the following command to create a record:
+
+```bash
+lxc network zone record create <network_zone> <record_name>
+```
+
+This command creates an empty record without entries and adds it to a network zone.
+
+#### Record properties
+
+Records have the following properties:
+
+Property          | Type       | Required | Description
+:--               | :--        | :--      | :--
+name              | string     | yes      | Unique name of the record
+description       | string     | no       | Description of the record
+entries           | entry list | no       | A list of DNS entries
+config            | string set | no       | Configuration options as key/value pairs (only `user.*` custom keys supported)
+
+### Add or remove entries
+
+To add an entry to the record, use the following command:
+
+```bash
+lxc network zone record entry add <network_zone> <record_name> <type> <value> [--ttl <TTL>]
+```
+
+This command adds a DNS entry with the specified type and value to the record.
 
-Each record has a name, a number of entries, an optional description and can also store user key/value config (`user.*`).
-Each entry is made of a type, value and optional TTL.
+For example:
+```bash
+lxc network zone record entry add <network_zone> <record_name> "A" "1.2.3.4"
+lxc network zone record entry add <network_zone> <record_name> "AAAA" "1234::1234"
+```
+You can use the `--ttl` flag to set a custom time-to-live for the entry.
+Otherwise, the default of 300 s is used.
 
-If you wanted to add a dual-stack web server, you may then create a record called `web`, set `Web server` as the description and then create two entries on it, one with type `A` and value `1.2.3.4` and the other with type `AAAA` and value `1234::1234`.
```

Comment:
Is this example of a common scenario not useful/relevant on how to use custom records, like the text around why adding DNS records is useful for SMTP servers above?

Reply:
This is the example I'm using for the custom records. But it definitely make sense to add that this will create a dual-stack web server. :)